pkg/console/transport/transport.go

```diff
@@ -0,0 +1,62 @@
+package transport
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"net/http"
+
+	"k8s.io/apimachinery/pkg/util/net"
+	ktransport "k8s.io/client-go/transport"
+)
+
+// TODO: this is borrowed from:
+//   https://github.com/openshift/cluster-authentication-operator/blob/df3160d1ab9f24448c853bfe567c44a2763017cb/pkg/operator2/transport.go
+//   but is intended to be moved to library-go at some point (see comments within the file)
+//   When that happens, remove this code & replace with the library-go version
+
+// transportFor returns an http.Transport for the given ca and client cert data (which may be empty)
+func transportFor(serverName string, caData, certData, keyData []byte) (http.RoundTripper, error) {
```

Comment:
Is this func actually called somewhere ?

Reply:
Wup, didn't intend to include this.  I'll remove.  Its potentially for a different PR.